### PR TITLE
Mayhem Integration

### PR DESCRIPTION
**Pull request by Gediminas19**

### CODE REVIEW

File: mayhem/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM fuzzers/cargo-fuzz:0.10.0
+
+ADD . /tokei
+WORKDIR /tokei
+RUN rustup toolchain install nightly
+WORKDIR /tokei/fuzz
+RUN RUSTFLAGS="-Znew-llvm-pass-manager=no" cargo +nightly fuzz build
+
+# Set to fuzz!
```
Comment (xansec) [inline mayhem/Dockerfile:9]: Is it possible to copy your targets from this build stage into a separate stage, to reduce image size?

**Review by xansec — CHANGES_REQUESTED** [review]
Unless you have changed the libraries/crates that the project is using, I would recommend not committing your `Cargo.lock` file. Please remove.

**Comment by xansec** [comment]
Please make the above requested changes, or we will have to close this PR

**Comment by Gediminas19** [comment]
Sorry, I'll do it tonight

**Comment by xansec** [comment]
It looks like the `Cargo.lock` changes are still in this PR. My guess is that since you've updated them, they're now tracking changes in this branch. You might want to create a new PR with only the 4 files you are interested in committing (you could create a new branch) and then create a PR with that new branch.

**Comment by xansec** [comment]
Awesome, LGTM!

**Pull request merged**